.github/workflows/ci.yml

```diff
@@ -4,6 +4,11 @@ on:
   push:
   pull_request:
   workflow_call:
+  # Run weekly Sundays at 12:37am
+  schedule:
+    - cron: "37 0 * * 0"
```

Comment:
I think your cron expression currently says: every sunday at 00:37am

Instead, I would propose  that it could run every saturday morning, for example:
`00 09 * * 6` https://crontab.guru/#00_09_*_*_6

...this would leave time to fix stuff on weekends.